consensus/vbft/block_pool.go

```diff
@@ -613,6 +620,9 @@ func (pool *BlockPool) addSignaturesToBlockLocked(block *Block, forEmpty bool) e
 					bookkeepers = append(bookkeepers, endoresrPk)
 					sigData = append(sigData, sig.Signature)
 				}
+				if block.CrossChainMsg != nil {
```

Comment:
add crossChainMsg.Sigdata when endroserPk is not nil, as above?

Reply:
updated